addons/mass_mailing/static/src/js/mass_mailing_snippets.js

```diff
@@ -145,6 +145,9 @@ options.registry.ImageOptimize.include({
      * @override
      */
     _getCSSColorValue(color) {
+        if (!color || ColorpickerWidget.isCSSColor(color)) {
```

Comment:
the `[]` are missing in the commit message
```
[FIX]: mass_mailing: fix traceback on picking shape
```
can you add the task id as well ?

Reply:
Ouch, was too hasty.